source/braille.py

```diff
@@ -808,6 +808,14 @@ def getFormatFieldBraille(field, isAtStart, formatConfig):
 				# Translators: Displayed in braille for a heading with a level.
 				# %s is replaced with the level.
 				textList.append(_("h%s")%headingLevel)
+	if formatConfig["reportLinks"]:
+		link=field.get("link")
+		oldLink=fieldCache.get("link") if fieldCache is not None else None

```

Comment:
1. This would make it impossible to tell the difference between an empty cache and no cache. The distinction is important because it determines whether you update the cache at the end.
2. And if you do update the cache at the end regardless, the empty dictionary you specified as a default would no longer be empty for future runs. That's a common Python gotcha. Consider the following code:

> ```
> def f(l=[]):
>    l.append("test")
>    print l
>
> f()
> f()
> ```

Here's the output:

> ~~~
> ['test']
> ['test', 'test']
> ```

In short, you absolutely shouldn't do this. :)